go/vt/vtgate/vstream_manager.go

```diff
@@ -575,15 +578,24 @@ func (vs *vstream) streamFromTablet(ctx context.Context, sgtid *binlogdatapb.Sha
 			})
 		}()
 
-		log.Infof("Starting to vstream from %s", tablet.Alias.String())
+		var options *binlogdatapb.VStreamOptions
+		const SidecardHeartbeatTableName = "heartbeat"
```

Comment:
I think we should keep the option open here to stream any table.
Other than that, should we declare the `const` outside of the method?